src/main/java/org/apache/cassandra/sidecar/routes/HealthService.java

```diff
@@ -53,18 +52,13 @@ public HealthService(CassandraAdapterDelegate cassandra)
     }
 
     @Operation(summary = "Health Check for Cassandra's status",
-    description = "Returns HTTP 200 if Cassandra is available, 503 otherwise",
-    responses = {
-    @ApiResponse(responseCode = "200", description = "Cassandra is available"),
-    @ApiResponse(responseCode = "503", description = "Cassandra is not available")
-    })
+    description = "Returns HTTP 200 if Sidecar is available")
```

Comment:
The changes in this file do not seem to be related with the patch?

Reply:
@yifan-c We wanted to avoid having sidecar's health response dependent on Cassandra's health status. Since the changes needed were minimal included these along with this patch